app/templates/components/side-bar.hbs

```diff
@@ -1,4 +1,7 @@
 <div class="ui left inverted vertical menu sidebar {{if this.sidebarVisible 'visible'}}">
+  <LinkTo
+    @route="pricing" class="item"
+    @activeClass="">{{t 'Pricing'}}</LinkTo>
```

Comment:
Not correct according to PR title

Reply:
these were the requested changes, what should be the PR title then?

Reply:
PR title says: pricing option added for anonymous users
Does this add pricing option for only anonymous users? Because the issue asks for that

Reply:
Issue resolved. Please review